types/xmlbuilder/index.d.ts

```diff
@@ -85,5 +85,5 @@ declare class XMLElementOrXMLNode {
 }
 
 declare namespace xmlbuilder {
-    function create(name: string, xmldec?: Object, doctype?: any, options?: Object): XMLElementOrXMLNode;
+    function create(nameOrObjSpec: string | { [name:string]: Object }, xmldec?: Object, doctype?: any, options?: Object): XMLElementOrXMLNode;
```

Comment:
If we choose `Object` as the type for the first argument then values of type `number`, `array`, `boolean` are also allowed. `create()`called with any of these values throws an exception.